### PR TITLE
Updated SBOM guide

### PR DESCRIPTION
**Pull request by sboldyreva**

### CODE REVIEW

**Comment by sboldyreva** [comment]
@javihernandez could you also take a look please

File: docs/documentation/sbom-guide.md [hunk 1]
```diff
@@ -2,425 +2,239 @@
...
-macOS are trademarks of Apple Inc., registered in the U.S. and other countries.
+If you have a question, want to file a bug, please, create an [issue](https://github.com/AlmaLinux/alma-sbom/issues) and reach out to us trough the [SIG/Build System ](https://chat.almalinux.org/almalinux/channels/build-system) chat channel.
 
-Windows is a registered trademark of Microsoft Corporation in the United States and other countries.
+Interested in contributing to the project?
+* Fork the [repository](https://github.com/AlmaLinux/alma-sbom).
+* Create a new feature branch.
+* Write your change.
+* Submit a pull request.
+* The AlmaLinux Team will appreciate and review your contribution!
```
Comment (bennyvasquez) [inline docs/documentation/sbom-guide.md:240]:
```suggestion
* The SIG members will appreciate and review your contribution! 
```

**Comment by bennyvasquez** [comment]
Just one thing here from me, but I didn't confirm any of the technical steps. This looks great!

File: docs/documentation/sbom-guide.md [hunk 2]
```diff
@@ -2,425 +2,239 @@
...
-To just query the *SBOM* records (versus creating them) one needs only to install the above mentioned `cas` tool. *The CAS* organization provides it in form of source and binary package(s) via their [github](https://github.com/codenotary/cas) and official web service
-
-In the [README.md](https://github.com/codenotary/cas/blob/master/README.md) you can find a cross-platform installation method, presented below.
+[AlmaLinux Build System](https://github.com/AlmaLinux/build-system) has implemented *SBOM* into its pipeline for security purposes, such as tracing the build process, ensuring that only trusted sources are used, and reducing the risk of data corruption. 
 
-&nbsp;
+## About immudb
 
-### Installation 
+The Codenotary [immudb](https://docs.immudb.io/master/immudb.html#what-is-immudb) database is integrated with AlmaLinux to ensure data integrity and provide a chain of trust and traceability. Each stage of the build process goes through authentication and notarization.
```
Comment (javihernandez) [inline docs/documentation/sbom-guide.md:17]: `... immudb databse is integrated with AlmaLinux` sounds weird to me. I'd say, and as a continuation of the previous sentence: `The AlmaLinux SBOM data is stored in Codenotary's immudb, which ensures data integrity and ...`

File: docs/documentation/sbom-guide.md [hunk 3]
```diff
@@ -2,425 +2,239 @@
...
+[AlmaLinux Build System](https://github.com/AlmaLinux/build-system) has implemented *SBOM* into its pipeline for security purposes, such as tracing the build process, ensuring that only trusted sources are used, and reducing the risk of data corruption. 
 
-&nbsp;
+## About immudb
 
-### Installation 
+The Codenotary [immudb](https://docs.immudb.io/master/immudb.html#what-is-immudb) database is integrated with AlmaLinux to ensure data integrity and provide a chain of trust and traceability. Each stage of the build process goes through authentication and notarization. 
 
-#### Official *CAS* installation method (*Linux®*, *macOS*)
+AlmaLinux OS Team has developed SBOM data management [utility](https://github.com/AlmaLinux/alma-sbom) that generates SBOM records and allows users to track a package to verify if it is notarized and trusted.
```
Comment (javihernandez) [inline docs/documentation/sbom-guide.md:19]: I suggest: `... has developed [alma-sbom](https://github.com/AlmaLinux/alma-sbom), the SBOM data management utility for AlmaLinux, that generates ...`

File: docs/documentation/sbom-guide.md [hunk 4]
```diff
@@ -2,425 +2,239 @@
...
-
-:::tip
-The *CAS* package is available in AlmaLinux Architect's COPR (Community Owned Package Repository).  Here, we install it on system based on [AlmaLinux 9.1](https://wiki.almalinux.org/release-notes/9.1.html) release.
-:::
-     
-##### 1. Enable the COPR repo for codenotary package
-```
-$ sudo dnf copr --hub build.almalinux.org enable andrewlukoshko/codenotary
-```
+To access the immudb SBOM information for AlmaLinux you need an internet connection as records are stored in the AlmaLinux `git` repository.
```
Comment (javihernandez) [inline docs/documentation/sbom-guide.md:25]: How about something like: `To access the SBOM information for AlmaLinux, you need an internet connection to retrieve the records from our AlmaLinux immudb instance.`

File: docs/documentation/sbom-guide.md [hunk 5]
```diff
@@ -2,425 +2,239 @@
...
+  ```
+* Activate the Virtual Environment:
+  ```
+  source env/bin/activate
+  ```
+* Install dependencies:  
+  ```
+  pip install .
+  ```
+  Among installed dependencies there's [immudb-wrapper](https://github.com/AlmaLinux/immudb-wrapper) that is used to get SBOM information based on the rpm package hash.
```
Comment (javihernandez) [inline docs/documentation/sbom-guide.md:41]: maybe something like: `there's immudb-wrapper, that is used to interact with our immudb database to save or get SBOM information`

**Comment by sboldyreva** [comment]
@javihernandez thank you for your review, applied your suggestions!

File: docs/documentation/sbom-guide.md [hunk 6]
```diff
@@ -14,15 +14,15 @@ title: "User Guide: AlmaLinux *Software Bill of Materials (SBOM)*"
 
 ## About immudb
 
-The Codenotary [immudb](https://docs.immudb.io/master/immudb.html#what-is-immudb) database is integrated with AlmaLinux to ensure data integrity and provide a chain of trust and traceability. Each stage of the build process goes through authentication and notarization. 
+The AlmaLinux SBOM data is stored in Codenotary's [immudb](https://docs.immudb.io/master/immudb.html#what-is-immudb), which ensures data integrity and provides a chain of trust and traceability. Each stage of the build process goes through authentication and notarization. 
 
-AlmaLinux OS Team has developed SBOM data management [utility](https://github.com/AlmaLinux/alma-sbom) that generates SBOM records and allows users to track a package to verify if it is notarized and trusted.
+AlmaLinux OS Team has developed has developed [alma-sbom](https://github.com/AlmaLinux/alma-sbom), the SBOM data management utility for AlmaLinux, that generates SBOM records and allows users to track a package to verify if it is notarized and trusted.
```
Comment (javihernandez) [inline docs/documentation/sbom-guide.md:19]: `has developed` is duplicated

**Review by javihernandez — COMMENTED** [review]
Thanks for the update @sboldyreva! Left a minor comment. Other than that, this looks great to me :)

**Comment by sboldyreva** [comment]
@javihernandez how could I miss that! thanks for noticing :)

**Review by javihernandez — APPROVED** [review]

**Review by bennyvasquez — APPROVED** [review]

**Pull request merged**